lib/conform.ex

```diff
@@ -73,10 +73,30 @@ defmodule Conform do
     end
   end
 
+  defp load_imports(opts) do
+    release_dir = Path.dirname(opts.schema)
+    {:ok, [_ | zip_files]} = :zip.list_dir(("#{release_dir}/app.schema.deps.ez") |> to_char_list)
```

Comment:
In conjunction, if there no imports and no archive created (  https://github.com/bitwalker/exrm/pull/160/files#diff-3af356a8acb8942aeae5ea2692f69360R48 ), what happens on release start with conform configuration, without archive, but assuming that it is always here?